assignment.js

```diff
@@ -113,6 +122,39 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const playerAChoice = "scissors"
+const playerBChoice = "paper"
+
+let compare = function(playerAChoice, playerBChoice) {
+    if(playerAChoice === playerBChoice) {
+        return "Game is a tie"
+    }
+
+    if(playerAChoice === "rock") {
+        if(playerBChoice === "scissors") {
+            return "playerA wins"
+        } else {
+            return "playerB wins"
+        }
+    }
+    if(playerAChoice === "paper") {
+        if(playerBChoice === "rock") {
+            return "playerA wins"
+        } else {
+            return"playerB wins"
+        }
+    }
+    if(playerAChoice === "scissors") {
+        if(playerBChoice === "paper") {
+            return "playerA wins"
+        } else {
+            return "playerB wins"
+        }
+    }
+}
+
+console.log(compare(playerAChoice, playerBChoice))
+
```

Comment:
@rlou88 Nice job wrapping a conditional statement in a function. I would suggest adding one more conditional that will check to see if there is an error is the player selections are wrong. Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L150)